data/LDS/projects.yaml

```
@@ -65,9 +65,17 @@ projects:
       - type: Paper
         title: "PriFi: Low-Latency Metadata Protection for Organizational Networks"
         url: https://arxiv.org/pdf/1710.10237.pdf
+        notes:
+          - label: Published at
+            text: "Proceedings on Privacy Enhancing Technologies (PoPETs)"
+            url: https://sciendo.com/article/10.2478/popets-2020-0061
       - type: Paper
         title: "PriFi: A Low-Latency and Tracking-Resistant Protocol for Local-Area Anonymous Communication"
-        url: http://www.cs.yale.edu/homes/jf/PriFi-WPES2016.pdf
+        url: https://infoscience.epfl.ch/record/223389?ln=en
```

no need to force language of the page.

```suggestion
        url: https://infoscience.epfl.ch/record/223389
```